metadata/pioneer.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>net.pioneerspacesim.Pioneer</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-only</project_license>
+	<name>Pioneer</name>
+	<summary>A game of lonely space adventure</summary>
+	<url type="homepage">https://pioneerspacesim.net/</url>
+	<url type="faq">https://pioneerwiki.com/wiki/FAQ</url>
+	<launchable type="desktop-id">pioneer</launchable>
+
+	<description>
+		<p>Pioneer is a space adventure game set in our galaxy at the
+			turn of the 31st century.
+		</p>
+		<p>The game is open-ended, and you are free to eke out whatever
+			kind of space-faring existence you can think of. Look
+			for fame or fortune by exploring the millions of star
+			systems. Turn to a life of crime as a pirate, smuggler
+			or bounty hunter. Forge and break alliances with the
+			various factions fighting for power, freedom or
```

Comment:
Actually, we just corrected this text on our website (source, it hasn't refreshed yet), to something that's more correct, since one can not "forge alliances with factions" in Pioneer. Not sure where that came from.
See:
https://github.com/pioneerspacesim/pioneer-web/pull/10/files

Sorry for slow progress, but pioneer development moves in glacial time.